.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: "cimg/elixir:1.13.3"
+    steps:
+      - checkout
+      - run: elixir ./lib/tic_tac_toe.ex
```

Comment:
Could you replace this with a `mix` command that achieves the same objective? What if the entry point of the application were to change from `tic_tac_toe.ex`?

Reply:
That is a good point that I didn't think of, will do!